### PR TITLE
MHV-42364 Added PDF download for EKG

### PR DESCRIPTION
**Pull request by Wyzardsleeves**
## Summary

- Added pdf download
- Not a bug. (Feature added)
- MHV Medical Records

## Related issue(s)

- [MHV-42364](https://jira.devops.va.gov/browse/MHV-42364)

## Testing done

- The download button now works to download a pdf.
- Migrate to an EKG entry under Labs and Tests and click the "download as PDF"

### CODE REVIEW

**Review by mmoyer-va — APPROVED** [review]

File: src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx [hunk 1]
```diff
@@ -9,7 +11,68 @@ const EkgDetails = props => {
 
   const formattedDate = formatDateLong(record?.date);
 
-  const download = () => {};
+  const generateEkgDetails = async () => {
+    const pdfData = {
+      headerLeft: 'Roberts, Jesse',
+      headerRight: 'Date of birth: January 1, 1970',
+      footerLeft: `Report generated by My HealtheVet and VA on ${moment().format(
```
Comment (acrollet) [inline src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx:18]: I think this is a different date format than other PDFs (`LL` vs `LLL`) - is that intentional?

File: src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx [hunk 2]
```diff
@@ -9,7 +11,68 @@ const EkgDetails = props => {
...
+            ],
+          },
+        ],
+      },
+    };
+
+    try {
+      await generatePdf('medicalRecords', 'electrocardiogram_report', pdfData);
+    } catch (error) {
+      // Error logging/presentation goes here...
```
Comment (acrollet) [inline src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx:69]: Let's make sure that any errors are getting reported to Sentry :-)

**Review by acrollet — CHANGES_REQUESTED** [review]
Just a couple little things to look at 🙂

File: src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx (hunk 1 above)
Comment (Wyzardsleeves) [reply src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx:19]: addressed

File: src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx (hunk 2 above)
Comment (Wyzardsleeves) [reply src/applications/mhv/medical-records/components/LabsAndTests/EkgDetails.jsx]: addressed

**Comment by Wyzardsleeves** [comment]
@acrollet hey did you see my changes?

**Review by acrollet — APPROVED** [review]
approved, thanks for the updates 🙂

**Pull request merged**